vite.config.js

```diff
@@ -43,4 +44,9 @@ export default defineConfig({
       url: "url-parse", // workaround for parse-link-header library that replaces nodejs builtin module with the module adapted for browser
     },
   },
+  test: {
+    environment: "jsdom",
+    globals: true,
```

Comment:
Just wondering ... do we still use globals ?

Reply:
@blcham  Not anymore, I will remove it